src/components/Dialog.jsx

```diff
@@ -15,4 +16,18 @@ const Dialog = ({
     </div>
 );
 
+Dialog.propTypes = {
+    open                : PropTypes.bool,
+    children            : PropTypes.any,
+    onContainerClick    : PropTypes.func,
+    className           : PropTypes.string,
+};
+
+Dialog.defaultProps = {
+    open                : PropTypes.bool,
```

Comment:
@shayanbinary `PropTypes.bool` is not default value for a propery.

Reply:
oops!